### PR TITLE
add delumen module

### PR DESCRIPTION
**Pull request by delumen-ivan**
please review my delumen module

### CODE REVIEW

File: errawrs/delumen.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+            elif choice == "4":
+                try:
+                    price = int(input("Enter the price: "))
+                    self.set_price(price)
+                except ValueError:
+                    print("Invalid input! Please enter a valid number for price.")
+
+            elif choice == "5":
+                try:
+                    year = int(input("Enter the price: "))
```
Comment (victorioso-daniel-2) [inline errawrs/delumen.py:67]: should be enter the year not price

File: errawrs/delumen.py [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+                    self.set_year(year)
+                except ValueError:
+                    print("Invalid input! Please enter a valid number for year.")
```
Comment (ghost) [inline errawrs/delumen.py]: exceeding column 80

File: errawrs/delumen.py [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+            elif choice == "3":
+                color = input("Enter the color: ")
+                self.set_color(color)
+
+            elif choice == "4":
+                try:
+                    price = int(input("Enter the price: "))
+                    self.set_price(price)
+                except ValueError:
+                    print("Invalid input! Please enter a valid number for price.")
```
Comment (ghost) [inline errawrs/delumen.py]: exceeding column 80

**Review by victorioso-daniel — CHANGES_REQUESTED** [review]
review your code
follow coding guidelines

File: errawrs/delumen.py (hunk 3 above)
Comment (delumen-ivan) [reply errawrs/delumen.py]: i already fix the code

File: errawrs/delumen.py (hunk 1 above)
Comment (delumen-ivan) [reply errawrs/delumen.py]: i already fix the code

File: errawrs/delumen.py (hunk 2 above)
Comment (delumen-ivan) [reply errawrs/delumen.py]: i already fix the code

Comment (delumen-ivan) [reply errawrs/delumen.py]: already fix

**Comment by delumen-ivan** [comment]
> review your code follow coding guidelines
all done

**Review by victorioso-daniel — APPROVED** [review]
LGTM

**Review by reyes-simonejake — APPROVED** [review]
LGTM

**Pull request merged**